client/state/themes/actions/activate-theme.js

```diff
@@ -39,6 +41,7 @@ export function activateTheme(
 				...( isEnabled( 'themes/theme-switch-persist-template' ) && {
 					persist_homepage_template: true,
 				} ),
+				...( styleVariationSlug && { style_variation_slug: styleVariationSlug } ),
```

Comment:
Note that we also need to update the activate theme endpoint on the Jetpack side to support style variation slug in this way.

Reply:
I have submitted a PR in Jetpack https://github.com/Automattic/jetpack/pull/28207. Do you know if updating the endpoint on the Jetpack side is enough for the parameter to passthrough to the WPCOM API, or do we need to re-implement the style variation setup on Jetpack's side as well?

Reply:
I think it might not be enough as [Jetpack_JSON_API_Themes_Active_Endpoint->switch_theme](https://github.com/Automattic/jetpack/blob/db731bb3cb13a76eae0fc301e8e6118e31823410/projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-themes-active-endpoint.php#L36) doesn't have any logic related to the style variation, and `wpcom_set_global_theme_style_variation` doesn't exist on the Jetpack site

Reply:
Seems that we will need to add `wpcom_set_global_theme_style_variation` to the wpcomsh repo 🤔